SmthGrey/config.ini

```diff
@@ -0,0 +1,21 @@
+# SmthGrey config.ini
+
+# For full descriptions of what each of these entries does,
+# along with the default values for each, see the Wiki page here:
+# https://github.com/profezzorn/ProffieOS/wiki/Sound-Font-config.ini-Configuration
+
+# humStart default = 100
+humStart=751
+volHum=15
```

Comment:
I think we should comment out all the variables below which have default values.
Makes it easy to see which ones actually do something.
We can also add a comment about it, something like:

All possible config variables listed below, but variables with default values are commented out.

Reply:
Done.